### PR TITLE
Fix port for healthcheck

### PR DESCRIPTION
**Pull request by JohnPreston**
Given that `PLATFORM_LISTENING_PORT` - soon to be replaced with `CDK_LISTENING_PORT` defaults to 8080, the healthcheck need to be updated to reflect that change.

PS: unsure if `CMD-SHELL` will use the env var substitution, but I think it does.

### CODE REVIEW

**Comment by JohnPreston** [comment]
Any feedback? @guizmaii @rokh-conduktor

File: doc/Public_API.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Could be used to setup probes on kubernetes or docker-compose.
 #### docker-compose probe setup
 ```yaml
 healthcheck:
-    test: ["CMD-SHELL", "curl --fail http://localhost/platform/api/modules/health/live"]
+    test: ["CMD-SHELL", "curl --fail http://localhost:${PLATFORM_LISTENING_PORT:-8080}/platform/api/modules/health/live"]
```
Comment (qboileau) [inline doc/Public_API.md:19]: I think you can use `CDK_LISTENING_PORT` directly as latest version support it (with fallback on `PLATFORM_LISTENING_PORT` also)

File: doc/Public_API.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ healthcheck:
 livenessProbe:
     httpGet:
         path: /platform/api/modules/health/live
-        port: http
+        port: http # Adapt based on PLATFORM_LISTENING_PORT
```
Comment (qboileau) [inline doc/Public_API.md:31]: should be defined in container port 
we can maybe add this for completness
```yaml
        ports:
          - containerPort: 8080
            protocol: TCP
            name: http 
         env: 
          - name: CDK_LISTENING_PORT
            value: 8080
```

Comment (JohnPreston) [reply doc/Public_API.md:31]: Right, I did not want to "override" `http` assuming it defaults to 80 when used (for K8s that is).

I suppose, either way, the healtcheck needs to point to the port defined by `CDK_LISTENING_PORT`

**Comment by JohnPreston** [comment]
Updated for your review @qboileau  ;)

**Review by qboileau — APPROVED** [review]
LGTM, thank you for your contribution.

**Pull request merged**